build/docker/zap_common.py

```diff
@@ -319,3 +332,47 @@ def zap_get_alerts(zap, baseurl, blacklist, out_of_scope_dict):
         alerts = zap.core.alerts(start=st, count=pg)
     logging.debug('Total number of alerts: ' + str(alert_count))
     return alert_dict
+
+
+def get_latest_zap_client_version():
+    version_info = None
+
+    try:
+        version_info = urlopen('https://pypi.python.org/pypi/python-owasp-zap-v2.4/json')
+    except Exception, e:
```

Comment:
This could be tweaked to support Python 3:
```
Traceback (most recent call last):
  File "zap-baseline.py", line 54, in <module>
    from zap_common import *
  File "/zaproxy/build/docker/zap_common.py", line 342
    except Exception, e:
                    ^
SyntaxError: invalid syntax
```